### ISSUE TITLE
Arteriovenous (AV) crossing sign

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10985]
Preferred term label:
Arteriovenous (AV) crossing sign

Synonyms
AV nicking, Arteriovenous nicking, AV crossing changes, Gunn’s sign

Definition (free text, please give PubMed ID)
A retinal vascular abnormality characterized by indentation or compression of a retinal vein at an arteriovenous crossing, leading to changes in venous caliber. AV crossing signs are commonly associated with hypertensive retinopathy and may contribute to retinal vein occlusion. They are indicative of vascular remodeling due to chronic hypertension or other systemic vascular diseases.

Parent term (use hpo.jax.org/app)
Abnormal retinal vascular morphology (HP:0008046)

Diseases characterized by this term
Hypertensive retinopathy, Retinal vein occlusion

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 

Adding new term: 
- ID: HP:6001216
- label: Arteriovenous crossing sign
- parents: [Abnormal retinal vascular morphology (HP:0008046)](https://hpo.jax.org/app/browse/term/HP:0008046)
- pmid: 
- def: A retinal vascular abnormality characterized by indentation or compression of a retinal vein at an arteriovenous crossing, leading to changes in venous caliber. AV crossing signs are commonly associated with hypertensive retinopathy and may contribute to retinal vein occlusion. They are indicative of vascular remodeling due to chronic hypertension or other systemic vascular diseases.
- comment: This feature may be present in diseases such as hypertensive retinopathy and retinal vein occlusion.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10985
- synonyms: AV crossing changes(NONE)|Gunn sign(NONE)|AV nicking(NONE)|Arteriovenous nicking(NONE)
